xml/decoders/TCS_X-Jan2000.xml

```diff
@@ -42,7 +42,7 @@
         <output name="Motor"/>
         <size length="3.03" width="0.355" height="0.135" units="inches"/>
       </model>
-      <model model="DP5X" numOuts="5" numFns="14" lowVersionID="30" highVersionID="40" maxMotorCurrent="1.3A (peak=2A)" formFactor="PnP">
+      <model model="DP5X" numOuts="5" numFns="14" maxMotorCurrent="1.3A (peak=2A)" formFactor="PnP">
```

Comment:
I know very little about these decoder definitions. But how is this decoder DP5X identified if it doesn't have lowVersionID and highVersionID? Just checking that it's correct.